front/app/containers/Admin/projects/edit/description/messages.ts

```diff
@@ -43,4 +43,17 @@ export default defineMessages({
     defaultMessage:
       'Decide on which message you want to give to your audience. Edit your project and enrich it with images, videos, file attachments,… This information helps visitors to understand what your project is about.',
   },
+  toggleLabel: {
+    id: 'app.containers.AdminPage.ProjectDescription.toggleLabel',
```

Comment:
Totally my bad that I didn't realize this before but these tranlsations need to be in the module, not here :)